pepper-apis/build-utils/build-support-images/Dockerfile-build

```diff
@@ -0,0 +1,67 @@
+# To make our config generation "compatible" with official dsdetoolbox, extract what is needed from it (this is not the base image)
+FROM broadinstitute/dsde-toolbox:DDO-1934 AS dsdetoolbox
+
+FROM maven:3.6.3-jdk-11 AS maven
+
+FROM google/cloud-sdk:alpine
+
+#Essentials for Maven
+COPY --from=maven /usr/share/maven /usr/share/maven
+RUN ln -s /usr/share/maven/bin/mvn /usr/bin/mvn
+COPY --from=maven /usr/local/bin/mvn-entrypoint.sh /usr/local/bin/mvn-entrypoint.sh
+COPY --from=maven /usr/share/maven/ref/settings-docker.xml /usr/share/maven/ref/settings-docker.xml
+
+ENV MAVEN_HOME /usr/share/maven
+ENV MAVEN_CONFIG "$USER_HOME_DIR/.m2"
+
+# Copying just configuration used by official DSDE
+COPY --from=dsdetoolbox /etc/consul-template/config/config.json /etc/consul-template/config/config.json
+ENV VAULT_ADDR https://clotho.broadinstitute.org:8200
+ENV CONSUL_TEMPLATE_VERSION=0.27.2
+ENV VAULT_VERSION=1.4.3
+
+# Set by build
+ARG TARGETARCH
+
+RUN case "${TARGETARCH:-amd64}" in \
+           arm64) export HASHICORP_ARCH='arm64' ;; \
+           amd64) export HASHICORP_ARCH='amd64' ;; \
+    esac; \
```

Comment:
This is the conditional that tells us which architecture this build is for. TARGETARCH variable is generated by Docker for us.